packages/zfpy/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: zfpy
+  version: 1.0.1
+  top-level:
+    - zfpy
+
+requirements:
```

Comment:
Let's rearrange the order of keys to package-source-build-requirements

Reply:
Addressed in 2253aa5202db3388aa0a5c46f0589ffe6d62f8b6.